keycloak/themes/haven/account/resources/css/haven.css

```diff
@@ -0,0 +1,281 @@
+html{
+    height: 100%;
+}
+
+body {
+  --mdc-theme-primary:#2FA09D;
+  --mdc-theme-accent:#E86953;
+  --mdc-theme-background: #f0f0f1;
+  display: flex;
+  flex-direction: column;
+  min-height: 100% !important;
+}
+
+header{
+    background-color: var(--mdc-theme-accent) !important;
+    color: var(--mdc-theme-text-primary-on-primary,#fff);
+}
+
+header > .mdc-toolbar__row{
+    min-height:64px;
+}
+
+header a{
+    color: var(--mdc-theme-text-primarymary-on-primary);
+}
+
+header a:hover, header a:focus{
+    color:#3f4145;
+    text-decoration: none;
+}
+
+h1.mdc-toolbar__title{
+    font-size: 20px;
+}
+
+.bs-sidebar{
+    background-color:#fff;
+    border-right-width: 2px;
+    width: 256px;
+}
+
+nav.mdc-list{
+    padding-top:0px;
+}
+
+nav.mdc-list > a{
+    font-family: serif !important;
+    font-size: 20px !important;
+    font-weight: 500 !important;
+    padding:29px 16px !important;
+}
+
+.bs-sidebar nav a.mdc-permanent-drawer--selected, .bs-sidebar nav a.mdc-permanent-drawer--selected:hover, .bs-sidebar nav a.mdc-permanent-drawer--selected:focus {
+    background-color:#888 !important;
+    color: #3F4145 !important;
+}
+
+.mdc-permanent-drawer .mdc-list-item__start-detail{
+    color:lightgray !important;
+}
+
+.bs-sidebar nav a.mdc-list-item:hover {
+    color: var(--mdc-theme-primary);
+}
+
+.mdc-permanent-drawer__content{
+    flex:1;
+}
+
+.drawer-logo{
+    text-align: center;
+    padding-bottom: 10px;
+}
+.content{
+    display: flex;
+    flex: 1 1 auto;
+    height: 100%;
+}
+
+.main{
+    width: 200vw;
+    background-color:var(--mdc-theme-background, #f0f0f1);
+    padding:30px;
+}
+
+h2{
+    font-family: 'Roboto', sans-serif !important;
+}
+
+a{
+    color:var(--mdc-theme-accent)
+}
+
+/* Material Icon Import */
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url(https://example.com/MaterialIcons-Regular.eot); /* For IE6-8 */
+  src: local('Material Icons'),
+    local('MaterialIcons-Regular'),
+    url(https://example.com/MaterialIcons-Regular.woff2) format('woff2'),

```

Comment:
...I didn't catch that hah. I'll figure /fix that. I'll need to check the instructions that MD offers but it will be done.